hack/run-wmcb-ci-e2e-test.sh

```diff
@@ -57,6 +57,19 @@ fi
 sed -i "s~ARTIFACT_DIR_VALUE~${ARTIFACT_DIR}~g" internal/test/wmcb/deploy/job.yaml
 sed -i "s~REPLACE_IMAGE~${WMCB_IMAGE}~g" internal/test/wmcb/deploy/job.yaml
 
+# declare required labels
+declare -a NS_LABELS=(
+  # turn on the automatic label synchronization required for PodSecurity admission
+  "security.openshift.io/scc.podSecurityLabelSync=true"
+  # set pods security profile to privileged. See https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels
+  "pod-security.kubernetes.io/enforce=privileged"
+)
+
+# apply required labels
+if ! $OC label ns default "${NS_LABELS[@]}" --overwrite; then
```

Comment:
Not sure about the implications of applying this to the `default` namespace. Should we have our own namespace?

Reply:
In the WMCO side we are using [wmco-test](https://github.com/openshift/windows-machine-config-operator/blob/6abe959a345258265ad4574bab39e48b2a2455d7/test/e2e/main_test.go#L109).

Reply:
Seems like a nit for now, as discussed it is fine to leave it as it is given this is a test only cluster.